common/cards/alter-egos/hermits/potatoboy-rare.ts

```diff
@@ -44,7 +44,7 @@ class PotatoBoyRare extends Card {
 		const {player} = component
 
 		observer.subscribe(player.hooks.onAttack, (attack) => {
-			if (!attack.isAttacker(component.entity) || attack.type !== 'secondary')
+			if (!attack.isAttacker(component.entity) || attack.type == 'secondary')
```

Comment:
Its best to use `===`. @Benji42 Would kill me if I didn't make you change this lol.

Reply:
I changed the change to be primary instead when I realized it might trigger on effect attacks.